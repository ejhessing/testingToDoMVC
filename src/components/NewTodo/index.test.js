import React from 'react';
import renderer from 'react-test-renderer';
import { StateContainer } from 'cerebral/react';

import NewTodo from '.';

describe('NewTodo component', () => {
  it('renders a form', () => {
    const state = { app: { newTodoTitle: 'Test title' } };
    const tree = renderer.create(<StateContainer state={state}><NewTodo /></StateContainer>);
    expect(tree).toMatchSnapshot();
  });
});
