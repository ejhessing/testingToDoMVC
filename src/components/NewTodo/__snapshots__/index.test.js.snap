exports[`NewTodo component renders a form 1`] = `
<form
  id="todo-form"
  onSubmit={[Function]}>
  <input
    autoComplete="off"
    className="new-todo"
    onChange={[Function]}
    placeholder="What needs to be done?"
    value="Test title" />
</form>
`;
